fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/tmc/LaneAreaDetector.java

```diff
@@ -57,6 +57,10 @@ public int getAmountOfVehiclesOnSegment() {
         return lastLaneAreaInfo == null ? 0 : Math.max(0, lastLaneAreaInfo.getVehicleCount());
     }
 
+    public int getAmountOfHaltingVehiclesOnSegment() {
```

Comment:
I see you adapted the name from  `getAmountOfVehiclesOnSegment`. However, since we can count the vehicles, we should rename them to `getNumberOf(Halting)VehiclesOnSegment`. Also, the term "segment" still originates from a specific usecase where we used lane-area-detectors as segments. I would therefore remove the "OnSegment" from both method names, thus calling them just `getNumberOf(Halting)Vehicles`.